compiler/src/iree/compiler/Dialect/Flow/TransformExtensions/FlowExtensionsOps.td

```diff
@@ -138,8 +141,9 @@ def ClonePrecedingOpIntoDispatchRegionOp : Op<
 }
 
 def MovePrecedingOpIntoDispatchRegionOp : Op<
-    Transform_Dialect, "iree.move_preceding_op_into_dispatch_region",
-    [TransformOpInterface]> {
+    Transform_Dialect, "iree.move_preceding_op_into_dispatch_region", [
+      // TODO: this is missing effects.
```

Comment:
Just the `MemoryEffectsOpInterface` declaration is missing in this line. The arguments and results have effect annotations.